pnpm-workspace.yaml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - www
+  - packages/*
+  - "!packages/create-slinkity"
```

Comment:
Hm, any reason we can't use `pnpm` on `create-slinkity`? I'd prefer to treat every `packages/` entry as a workspace if possible.